kalite/distributed/templates/distributed/loadtesting/load_test.html

```diff
@@ -268,16 +268,19 @@
                     this.goto("/learn/khan/math/arithmetic/addition-subtraction/basic_addition/addition_1/", callback);
                 },
                 function(input, callback) {
-                    var numbers = this.$("span[class=mord][style]")
-                    var answer = 0
+                    var numbers = this.$("span.mn");
```

Comment:
This won't work on new exercises, needs to be span.mord, or as above before it was changed.